Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -0,0 +1,166 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Since this is just another Settings view controller, and since there are no other classes in the `AuthenticationManager` group you're creating, is there any reason you don't want to just put this in Client > Frontend > Settings with the others?